doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -89,78 +194,291 @@ Add official PostgreSQL repository.
    $ sudo netstat -ntlp | grep 5432
    ```
 
-   For more details, please refer to the [official page](https://wiki.postgresql.org/wiki/Apt).  
+   For more details, please refer to the [official page](https://wiki.postgresql.org/wiki/Apt).
 
    Connect to the postgresql-server via the psql utility by executing the following command.
 
    ```bash
    $ sudo -u postgres psql
    ```
+
    Specify the password for the postgres user (the default password in Kaa configuration files is "admin").
 
    ```bash
-   postgres=# \password
-   Enter new password: admin
-   Enter it again: admin
+   postgres=# \password Enter new password: admin Enter it again: admin
    ```
 
    Create the Kaa database by executing the following command.
 
-   ```bash
+   ```sql
    CREATE DATABASE kaa;
    ```
 
-4. Install [MariaDB 5.5](https://mariadb.org/download/) (Optional, you may install [PostgreSQL 9.4](http://www.postgresql.org/download/) instead).
 
-   Install MariaDB 5.5 for Ubuntu 14.04 64-bit.
+
+5. Install [Zookeeper 3.4.5](http://zookeeper.apache.org/doc/r3.4.6/).
 
    ```bash
-    sudo apt-get install software-properties-common
-    sudo apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xcbcb082a1bb943db
-    sudo add-apt-repository 'deb [arch=amd64,i386] http://ftp.hosteurope.de/mirror/mariadb.org/repo/5.5/ubuntu trusty main'
+   $ sudo apt-get install zookeeperd
    ```
 
-   Once the key is imported and the repository added you can install MariaDB with:
+   You can check if the Zookeeper service is running by executing the following command.
 
    ```bash
-    $ sudo apt-get update
-    $ sudo apt-get install mariadb-server
+   $ netstat -ntlp | grep 2181
    ```
 
-   During the installation process you will be asked to configure the root password for the MariaDB, enter "admin".
 
-   ![alt text](mariadbpassword.png "mariadb installation password promt")
 
-   You can check if the MariaDB server is running by executing the following command.
+6. Install [MongoDB](http://www.mongodb.org/downloads) (Optional, you may install [Cassandra 3.5](http://cassandra.apache.org/download/) instead) ([source](http://docs.mongodb.org/v2.6/tutorial/install-mongodb-on-red-hat/)).
```

Comment:
Same notes as for SQL DBs. You should mention that user has option to install one of two NoSQL DBs. In addition this step can be presented as two tabs - one for MiongoDB, another one for Cassandra.